Adding the PB Piva, a Small  and Agile Cargo Ship!

Rubeebeebee opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->
Adding the PB piva, a small and agile cargo ship with a sideway shutter door for quick cargo transfer! 
Commissioned by @.catmage at discord


FTL capable, and runs on uranium. 

Costs only 21k for the whole package. 

Basically made to act as a getaway vehicle for the Paycheck Bratva, or any unscrupulous groups.  
## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
<img width="910" height="615" alt="piva armaments" src="https://github.com/user-attachments/assets/36ae9232-69f1-4787-b416-e92bf4e46f99" />
no ship guns to speak of, just three improvised shotguns and two shotgun shell boxes, 1x buckshot, 1x beanbag. 


## How to test
<!-- Describe the way it can be tested -->
Load the .yml up in an IDE, should work!

## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc). 
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->
<img width="627" height="786" alt="piva live" src="https://github.com/user-attachments/assets/2dcce47a-449b-4550-ada6-48c8fa4d3ea8" />
<img width="647" height="820" alt="piva" src="https://github.com/user-attachments/assets/36b2a0e9-e7bc-429e-abd3-696847e68d07" />
<img width="1149" height="202" alt="piva appraisal" src="https://github.com/user-attachments/assets/f01f1059-4f9e-4921-b080-26e83893ab18" />



## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.
- [X] I can confirm this PR contains no AI-generated content, and did not use any AI-generated content.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->

:cl:
- add: adding the PB Piva, a small and agile cargo ship!

Rubeebeebee committed:
adding the PB Piva, a small  and agile cargo ship!

monolith8319 committed:
chore: automatically update REUSE headers

UnicornOnLSD commented:
That's a pretty sick shuttle, yo.

grandalff commented:
Does Pluto not do this well doe

Rubeebeebee commented:
> Does Pluto not do this well doe

pretty much, but this ships got a side shutter doorway for easy space access.

UnicornOnLSD commented:
Hey, maker of the DIS Pluto here.

Pluto fills the niche indeed.

However I was working on a PBr version of the Pluto but have been waiting on more PBr assets to be added.

Hit me up Rubee if you want to try and make a custom version of the Pluto, I dig the left side blast door idea.

ALSO HAVING MODIFIED VERSION OF EXITISTING SHIPS FOR SPECIFIC GROUPS AND FACTIONS WOULD SOMEWHAT AVOID BLOAT

Rubeebeebee commented:
> Hey, maker of the DIS Pluto here.
> 
> Pluto fills the niche indeed.
> 
> However I was working on a PBr version of the Pluto but have been waiting on more PBr assets to be added.
> 
> Hit me up Rubee if you want to try and make a custom version of the Pluto, I dig the left side blast door idea.
> 
> ALSO HAVING MODIFIED VERSION OF EXITISTING SHIPS FOR SPECIFIC GROUPS AND FACTIONS WOULD SOMEWHAT AVOID BLOAT

ohh, sounds dope! maybe in the future if i get another idea down the line. This was for a commission so i wanted to make it unique

OneZerooo0 commented:
yeah, we already have the pluto, sorry.
Thanks for the contribution!

Rubeebeebee commented:
> yeah, we already have the pluto, sorry. Thanks for the contribution!

aw man!

Redrover1760 reviewed (commented):
This ship fills the niche of "even more tinier barren variant than Pluto" its not the same honestly. Design is different and thus this should actually be reviewed.

Redrover1760 committed:
Update piva.yml

Redrover1760 commented:
Price should be pretty significant cause it has ftl. Makes me think we need to relook at ftl pricing on general honestly.

UnicornOnLSD commented:
Price formula moment but yeah it forces any ship with ftl no matter what to be priced above 25k

Rubeebeebee committed:
upped the price to 25k

Rubeebeebee committed:
upped the price to 25k

Rubeebeebee committed:
uh... updated the piva to 25k

monolith8319 committed:
chore: automatically update REUSE headers

Rubeebeebee commented:
> Price should be pretty significant cause it has ftl. Makes me think we need to relook at ftl pricing on general honestly.

right, updated the price to 25k, as per @UnicornOnLSD 's suggestion. 

I could also remove the FTL drive entirely, and make the ship even cheaper. Makes for a really barebones cargoship that has to drive to depots manually.

Rubeebeebee commented:
> > Price should be pretty significant cause it has ftl. Makes me think we need to relook at ftl pricing on general honestly.
> 
> right, updated the price to 25k, as per @UnicornOnLSD 's suggestion.
> 
> I could also remove the FTL drive entirely, and make the ship even cheaper. Makes for a really barebones cargoship that has to drive to depots manually.

@Redrover1760

Redrover1760 commented:
> > > Price should be pretty significant cause it has ftl. Makes me think we need to relook at ftl pricing on general honestly.
> > 
> > 
> > right, updated the price to 25k, as per @UnicornOnLSD 's suggestion.
> > I could also remove the FTL drive entirely, and make the ship even cheaper. Makes for a really barebones cargoship that has to drive to depots manually.
> 
> @Redrover1760

honestly ftl removal would be ideal

super cheap entry level cargo ship is great

Rubeebeebee committed:
removed the FTL from the piva and changed the price to 17k

Rubeebeebee committed:
Merge branch 'Piva-Branch' of https://github.com/Rubeebeebee/Monolith into Piva-Branch

Rubeebeebee commented:
> > > > Price should be pretty significant cause it has ftl. Makes me think we need to relook at ftl pricing on general honestly.
> > > 
> > > 
> > > right, updated the price to 25k, as per @UnicornOnLSD 's suggestion.
> > > I could also remove the FTL drive entirely, and make the ship even cheaper. Makes for a really barebones cargoship that has to drive to depots manually.
> > 
> > 
> > @Redrover1760
> 
> honestly ftl removal would be ideal
> 
> super cheap entry level cargo ship is great

good news! ive removed the FTL and lowered the price to 17k after removing stuff stuff alongside the FTL, namely the SMES
<img width="1168" height="219" alt="supercheap piva" src="https://github.com/user-attachments/assets/c2a1e216-fb19-4316-9b03-68c8e93a31a1" />
<img width="706" height="840" alt="cheaper piva" src="https://github.com/user-attachments/assets/3b99d8e6-f717-43c5-92be-d2f80b5fbb88" />

Rubeebeebee committed:
updated the piva to have a fax, holopad, and defib.

also updated the price to 16500 because it somehow became cheaper after i removed the gun locker.

Rubeebeebee commented:
updated the piva to have a holopad, fax, and defib. 

Removed the gunlocker and transfered it's contents to the secure closet, weirdly enough this reduced its price back down to 16300, so adjusted the price to be 16500 instead. 

<img width="704" height="823" alt="newer piva" src="https://github.com/user-attachments/assets/b25c7cf4-f535-4cd7-8282-7523556876f6" />
<img width="1196" height="214" alt="newer piva appraisal" src="https://github.com/user-attachments/assets/a1569518-9329-48c7-b7fa-8a37e812ee95" />

monolith8319 committed:
chore: automatically update REUSE headers

Rubeebeebee commented:
@Redrover1760 THANK YOUUU
